src/routes/configuration.py

```diff
@@ -0,0 +1,115 @@
+from fastapi import APIRouter
+from starlette.status import HTTP_200_OK, \
+    HTTP_201_CREATED, HTTP_400_BAD_REQUEST, \
+    HTTP_404_NOT_FOUND
+
+from src.interfaces import ConfigurationInterface
+from src.models import Configuration, NewConfiguration
+from src.utils.messages import ConfigurationMessage
+from src.utils.encoder import BsonObject
+from src.utils.response import UJSONResponse
+
+
+configuration_routes = APIRouter(tags=["Configuration"])
+
+
+@configuration_routes.get("/configuration")
+def find_all():
+    """
+    Get all configuration exist in db
```

Comment:
Change: "Get all configurations exists in db" ---> "Get all existing configurations in db"